train.py

```diff
@@ -95,6 +105,14 @@ def train(model, dataset, data_csv, binary, binary_label, params_file, augmentat
             logger.error(f"Error while reading YAML: {exc}")
             raise exc
 
+
+    # hyperparameters:
+    N_EPOCHS = int(params[MODEL_NAME]['N_EPOCHS'])
+    BATCH_SIZE_TRAIN = int(params[MODEL_NAME]['BATCH_SIZE_TRAIN'])
+    BATCH_SIZE_TEST = int(params[MODEL_NAME]['BATCH_SIZE_TEST'])
+    OPTIMIZER = params[MODEL_NAME]['OPTIMIZER']
+    LR = float(params[MODEL_NAME]['LR'])
+    WEIGHT_DECAY = float(params[MODEL_NAME]['WEIGHT_DECAY'])
```

Comment:
```suggestion
```

Reply:
These come later at lines 211-216 in the else-branch for NN models. BOVW-model doesn't have all of these hyperparameters so they can't be defined here.